php/class-plugin.php

```diff
@@ -43,7 +43,7 @@ public function init() {
 	public function enqueue_editor_assets() {
 		wp_enqueue_script(
 			'wp-foo-bar-js',
-			$this->asset_url( 'assets/js/editor.js' ),
+			$this->asset_url( 'assets/js/block-editor.js' ),
```

Comment:
Nice catch. There's been a lot that has changed glad someone is paying attention.

Reply:
Yes I am carrying on testing it today.